### PR TITLE
VCF4.3 to VCF4.4 change list

### PR DESCRIPTION
**Pull request by tcezard**
I was asked to summarise the changes introduced between 4.3 and 4.4 and though I might as well add it here. 
I also added the change list between 4.1 and 4.2 for completeness

### CODE REVIEW

**Review by d-cameron — CHANGES_REQUESTED** [review]
Sorry about the delay in reviewing.

File: VCFv4.3.tex [hunk 1]
```diff
@@ -2150,4 +2150,10 @@ \subsection{Changes between BCFv2.1 and BCFv2.2}
 \item The implicit filter PASS was described inconsistently throughout BCFv2.1: It is encoded as the first entry in the dictionary, not the last.
 \end{itemize}
 
+\subsection{Changes between VCFv4.1 and VCFv4.2}
+\begin{itemize}
```
[the remark this answers is not in the document]
Comment (tcezard) [reply VCFv4.3.tex:2154]: I missed that. Thank you.

File: VCFv4.4.draft.tex [hunk 2]
```diff
@@ -2251,8 +2250,11 @@ \subsection{BCF2 block gzip and indexing}
 \section{List of changes}
 
 \subsection{Changes between VCFv4.4 and VCFv4.3}
+
 \begin{itemize}
-\item Started new Draft Spec
+\item Clarify how to add multiple SV per record by making the number of value in reserved SV INFO fields dependent on the number of ALT
+\item Deprecate SVTYPE INFO field preferring the use of symbolic alleles in the ALT field
+\item Define new reserved INFO field EVENT, EVENTTYPE and SVCLAIM
```
[the remark this answers is not in the document]
Comment (tcezard) [reply VCFv4.4.draft.tex:2257]: Added.

File: VCFv4.4.draft.tex [hunk 3]
```diff
@@ -2251,8 +2250,11 @@ \subsection{BCF2 block gzip and indexing}
 \section{List of changes}
 
 \subsection{Changes between VCFv4.4 and VCFv4.3}
+
 \begin{itemize}
-\item Started new Draft Spec
+\item Clarify how to add multiple SV per record by making the number of value in reserved SV INFO fields dependent on the number of ALT
```
[the remark this answers is not in the document]
Comment (tcezard) [reply VCFv4.4.draft.tex:2255]: Added

**Pull request merged**